src/styles/card.scss

```diff
@@ -0,0 +1,73 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  width: $cardWidth;
+  padding: 30px 14px 15px;
+
+  &:hover {
+    box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 45px;
+  }
+
+  &__description {
+    @include fontStyleNormall;
+    margin-bottom: 5px;
+  }
+
+  &__code {
+    margin-bottom: 15px;
+  }
+
+  &__rate,
+  &__price {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__rate {
+    align-items: center;
+  }
+
+  &__price {
+    margin-top: 20px;
+    align-items: baseline;
+  }
+
+  &__price--cost {
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 19px;
+  }
+
+  &__link {
+    background: $colorActive;
```

Comment:
link is not active, why colorActive?
I guess that's not a good name for the var